docs/user-guide/rules/validate-set-cookie-header.md

```diff
@@ -0,0 +1,179 @@
+# Validate `Set-Cookie` Header (`validate-set-cookie-header`)
+
+This rule validates the `set-cookie` header and confirms that the
+`Secure` and `HttpOnly` directives are defined when sent from a
+secure origin (HTTPS).
```

Comment:
Maybe we should rename this rule to something else since we are only looking for some specific directives. 

Nothing comes to mind, but maybe something in line lines of `secure-cookies` or `no-insecure-cookies`?